test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -335,4 +336,68 @@ def add_locations
     assert_response :success
     assert_equal 1, location.reload.location_parameters.count
   end
+
+  test "should create with comma separated name" do
+    name = "%s, %s" % [RFauxFactory.gen_alpha, RFauxFactory.gen_alpha]
+    post :create, params: { :location => { :name => name } }
+    assert_response :success
```

Comment:
also note that it would  be good if the test uses random name for each run to report what names were used in case of failure, how about this?

```ruby
assert_response :success, "creation with name #{name} failed with code #{response.code}"
```

Reply:
the same applies in the test below